gitlab/mixins.py

```diff
@@ -653,6 +653,62 @@ def download(
         )
 
 
+class RotateMixin(_RestManagerBase):
+    _computed_path: Optional[str]
+    _from_parent_attrs: Dict[str, Any]
+    _obj_cls: Optional[Type[base.RESTObject]]
+    _parent: Optional[base.RESTObject]
+    _parent_attrs: Dict[str, Any]
+    _path: Optional[str]
+    gitlab: gitlab.Gitlab
+
+    @exc.on_http_error(exc.GitlabRotateError)
+    def rotate(
+        self, id: Union[str, int], expires_at: Optional[str] = None, **kwargs: Any
+    ) -> Dict[str, Any]:
+        """Rotate an access token.
+
+        Args:
+            id: ID of the token to rotate
+            **kwargs: Extra options to send to the server (e.g. sudo)
+
+        Raises:
+            GitlabAuthenticationError: If authentication is not correct
+            GitlabRotateError: If the server cannot perform the request
+        """
+        data = {"expires_at": expires_at}
```

Comment:
Is it okay for this to be set to `None`?

Reply:
I tested this before making the PR and it works, but maybe we shouldn't send null parameters if the user doesn't send them just to keep requests cleaner. I changed this in https://github.com/python-gitlab/python-gitlab/commit/4f27e365de83e92804dbbfcc075f4b3c8a7a15dd, thanks!